arches_component_lab/src/arches_component_lab/widgets/ConceptMultiselectWidget/components/ConceptMultiselectWidgetEditor.vue

```diff
@@ -0,0 +1,123 @@
+<script setup lang="ts">
+import { ref, watchEffect } from "vue";
+import type { Ref } from "vue";
+
+import { useGettext } from "vue3-gettext";
+import TreeSelect from "primevue/treeselect";
+
+import { fetchConceptsTree } from "@/arches_component_lab/datatypes/concept/api.ts";
+import type { ConceptListValue } from "@/arches_component_lab/datatypes/concept/types.ts";
+
+import type { FormFieldResolverOptions } from "@primevue/forms";
+
+import type {
+    CollectionItem,
+    ConceptFetchResult,
+} from "@/arches_component_lab/datatypes/concept/types.ts";
+import GenericFormField from "@/arches_component_lab/generics/GenericFormField.vue";
+import { convertConceptOptionToFormValue } from "@/arches_component_lab/datatypes/concept/utils.ts";
+import type { AliasedNodeData } from "@/arches_component_lab/types.ts";
+
+const props = defineProps<{
+    graphSlug: string;
+    nodeAlias: string;
+    value?: ConceptListValue | string[] | null | undefined;
```

Comment:
I think we're trying to make value required and not allow anything other than the desired type.  EG:

```suggestion
    value: ConceptListValue;
```

Reply:
So is the intent that if you're using a widget without a resource (or node value) that "null" (or some empty value) must be provided to it the widget?

Reply:
As in, `ConceptListValue` should already be a union with `null`, and if not, we should fix it.